ArduCopter/Parameters.cpp

```diff
@@ -1082,7 +1105,7 @@ const AP_Param::ConversionInfo conversion_table[] = {
     { Parameters::k_param_arming_check_old,   0,      AP_PARAM_INT8,  "ARMING_CHECK" },
     // battery
     { Parameters::k_param_fs_batt_voltage,    0,      AP_PARAM_FLOAT,  "BATT_LOW_VOLT" },
-    { Parameters::k_param_fs_batt_mah,        0,      AP_PARAM_FLOAT,  "BATT_LOW_MAH" },
+  //  { Parameters::k_param_fs_batt_mah,        0,      AP_PARAM_FLOAT,  "BATT_LOW_MAH" },
```

Comment:
Yes, that needs to be removed because I've used its slot to create a new param. It's been deprecated, so shouldn't be an issue. I do need to remove the commented line though